tests/testthat/_snaps/uncount.md

```diff
@@ -1,9 +1,23 @@
-# errors on negative weights
+# validates inputs
 
     Code
-      (expect_error(uncount(df, w)))
-    Output
-      <error/rlang_error>
+      uncount(df, y)
+    Condition
       Error in `uncount()`:
-      ! all elements of `weights` must be >= 0
+      ! Can't convert `weights` <character> to <integer>.
+    Code
+      uncount(df, w)
+    Condition
+      Error in `uncount()`:
+      ! `weights` must be a vector of positive numbers. Location 1 is negative.
+    Code
+      uncount(df, x, .remove = 1)
+    Condition
+      Error in `uncount()`:
+      ! `.remove` must be `TRUE` or `FALSE`, not the number 1.
+    Code
+      uncount(df, x, .id = "")
+    Condition
+      Error in `uncount()`:
+      ! `.id` must be a valid name or `NULL`, not the empty string "".
```

Comment:
I noticed in stringr error messages you backtick the `""` but we didn't do that in the rlang check functions